php/libraries/NDB_BVL_Instrument_LINST.class.inc

```diff
@@ -446,49 +446,61 @@ class NDB_BVL_Instrument_LINST extends \NDB_BVL_Instrument
                             'instrument_title',
                             $pieces[1]
                         );
-                        if ($this->DataEntryType!=="DirectEntry") {
-                            $this->_addMetadataFields();
-                        }
                     }
-                    $this->dictionary = array_merge(
-                        $this->dictionary,
-                        [
-                            new DictionaryItem(
-                                $this->testName.'_Date_taken',
-                                'Date of Administration',
-                                $scope,
-                                new \LORIS\Data\Types\DateType(),
-                                new Cardinality(Cardinality::SINGLE),
-                                'Date_taken',
-                            ),
-                        ]
-                    );
-                    if (strrpos($this->testName ?? '', '_proband') === false) {
-                        if (!$this->postMortem) {
-                            $this->dictionary = array_merge(
-                                $this->dictionary,
-                                [
-                                    new DictionaryItem(
-                                        $this->testName.'_Candidate_Age',
-                                        'Candidate Age (Months)',
-                                        $scope,
-                                        new \LORIS\Data\Types\Duration(),
-                                        new Cardinality(Cardinality::SINGLE),
-                                        'Candidate_Age',
-                                    ),
-                                ]
-                            );
-                        } else {
+                    if ($this->DataEntryType!=="DirectEntry") {
```

Comment:
```suggestion
                    // The sequence of if statements below building the data dictionary is designed to
                    // mimic the same logic in the _addMetadataFields() function of
                    // NDB_BVL_Instrument to keep the dictionary in sync with the fields added 
                    // to the form, make sure to modify it accordingly
                    if ($this->DataEntryType!=="DirectEntry") {
```

Reply:
@ridz1208 -- think you have the wrong person!

Reply:
@skarya22 I think he means you.